addons/l10n_multilang/models/l10n_multilang.py

```diff
@@ -151,26 +151,29 @@ class BaseLanguageInstall(models.TransientModel):
 
     def lang_install(self):
         self.ensure_one()
-        already_installed = self.lang in [code for code, _ in self.env['res.lang'].get_installed()]
         res = super(BaseLanguageInstall, self).lang_install()
-        if already_installed:
+        lang_codes = set([lang.code for lang in self.lang_ids])
+        installed = {code for code, _ in self.env['res.lang'].get_installed()}
```

Comment:
`_` is used for translation, so in Odoo it's better to use double underscore for unused variable (to avoid to overwrite `_`)
```suggestion
        installed = {code for code, __ in self.env['res.lang'].get_installed()}
```